### PR TITLE
French (edits)

### PR DESCRIPTION
**Pull request by vhf**
This PR fixes some wording issues that were present in the French version.

I'll happily motivate these changes upon request.

### CODE REVIEW

**Comment by triblondon** [comment]
Need a native speaker review.  @kaelig?

File: index.html [hunk 1]
```diff
@@ -214,29 +214,29 @@ <h2>On behalf of organisations</h2>
...
 				</ol>
 
-				<p>Nous ne cherchons pas à stopper le développement de AMP par Google, et ces changements ne le requièrent pas. Aussi, nous approuvons la promotion par les moteurs de recherche des pages performantes. AMP peut demeurer l’une des technologies qui donnent aux éditeurs le choix entre une variété d’options de grande qualité afin de servir des pages Web rapidement et de rendre leur utilisation plus satisfaisante.</p>
+				<p>Nous ne cherchons pas à stopper le développement de AMP par Google, et ces changements ne le requièrent pas. Aussi, nous approuvons la promotion par les moteurs de recherche des pages performantes. AMP peut demeurer l’une des technologies qui donnent aux éditeurs le choix entre une variété d’options de grande qualité afin de servir des pages Web rapidement et de rendre leur utilisateurs plus satisfaits.</p>
```
Comment (kaelig) [inline index.html]: 👎

Less inclusive and introduces a grammar error.

File: index.html [hunk 2]
```diff
@@ -214,29 +214,29 @@ <h2>On behalf of organisations</h2>
...
-					<li>Ne pas intégrer de contenus tiers externes dans une page Google, à moins qu’il ne soit clair pour l’utilisateur ou l’utilisatrice d’avoir affaire à un produit Google. Il est parfaitement acceptable que Google lance un “lecteur de nouvelles”, mais il n’est ni acceptable de servir une page n’affichant que du contenu tiers depuis une URL de Google, ni d’obliger le tiers à utiliser l’hébergement de Google pour apparaître plus en évidence dans les résultats de recherche.</li>
+					<li>Ne pas intégrer de contenus tiers externes dans une page Google, à moins qu’il ne soit clair pour l’utilisateur ou l’utilisatrice d’avoir affaire à un produit Google. Il est parfaitement acceptable que Google lance un “lecteur de nouvelles”, mais il n’est ni acceptable de servir une page n’affichant que du contenu tiers depuis une URL de Google, ni d’obliger le tiers à utiliser l’hébergement de Google pour apparaître dans les résultats de recherche.</li>
```
Comment (kaelig) [inline index.html:534]: Good catch, this wasn’t changed when the English version was updated.

**Review by kaelig — CHANGES_REQUESTED** [review]
Looks great, just a small change and we’re good :)

File: index.html (hunk 1 above)
Comment (vhf) [reply index.html]: I agree on inclusiveness. No grammar error IMO, and semantics is closer to `making users happy`. I reverted (and rebased) anyway. :)

**Review by kaelig — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (kaelig) [reply index.html]: There was a _faute d'accord_: "leur utilisateurs plus satisfaits" -> leur**s**

**Pull request merged**